docs/docs/12-context.md

```diff
@@ -64,7 +64,7 @@ var Button = React.createClass({
   },
   render: function() {
     return (
-      <button style={{'{{'}}background: this.context.color}}>
+      <button style={{background: this.context.color}}>
```

Comment:
@zpao Sorry, I did not know that `{{`  is a Jekyll's opening tag. Thank you for it. And I  already reverted the change.